pymagicc/io.py

```diff
@@ -618,14 +618,19 @@ def read_data_block(self):
             region = convert_magicc_to_openscm_regions(self._stream.readline().strip())
 
             try:
-                variables = convert_magicc6_to_magicc7_variables(
-                    self._read_data_header_line(self._stream, "YEARS")
-                )
+                pos_vars = self._stream.tell()
+                variables = self._read_data_header_line(self._stream, "YEARS")
             except IndexError:  # tried to get variables from empty string
                 break
             except AssertionError:  # tried to get variables from a notes line
-                break
+                try:
+                    # just in case...
```

Comment:
Can you elaborate on the "just in case"? :smile:

Reply:
probably should read, `# check that the header line isn't labelled with "YEAR" instead` although I think there's a better way to do this so will refactor in 20 mins